Gemfile

```diff
@@ -79,14 +79,12 @@
 # the new version. It is always preferable to upgrade our code.
 source 'https://rubygems.org'
 
-ENV['RAILS_UPGRADE'] = 'true'
```

Comment:
Unintentional?

Reply:
Ah commits are showing in the wrong order. This happens after https://github.com/mysociety/alaveteli/pull/5748/commits/891dc4f8483104ab91381cc4c0fe1a9af20ddca3